.github/workflows/build-v2.1.yml

```
@@ -199,6 +199,123 @@
+          - [Sailfish.TestAdapter v${VERSION}](https://www.nuget.org/packages/Sailfish.TestAdapter/${VERSION})
+          - [Sailfish.Analyzers v${VERSION}](https://www.nuget.org/packages/Sailfish.Analyzers/${VERSION})
+          EOF
+
+          echo "Manual release notes prepared:"
+          cat release_notes.md
+
+          # Clean up the manual notes file after use
+          rm RELEASE_NOTES.md
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)